src/go/rdctl/cmd/snapshotCreate.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/snapshot"
+	"github.com/spf13/cobra"
+)
+
+var snapshotCreateCmd = &cobra.Command{
+	Use:   "create <name>",
+	Short: "Create a snapshot",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		paths, err := paths.GetPaths()
+		if err != nil {
+			return fmt.Errorf("failed to get paths: %w", err)
+		}
+		manager := snapshot.NewManager(paths)
+		if _, err := manager.Create(args[0]); err != nil {
```

Comment:
We should impose a limit on how long a snapshot name can be. Maybe 80 characters. or 100. But something...

Also you might want to do a regexp check on the contents of the proposed name, at least forbidding white-space and possibly other shell metacharacters (`!$^&*()[]{};:?/\\` and the three quote characters).